lib/MetaCPAN/Script/Mapping.pm

```diff
@@ -9,6 +9,18 @@ use MetaCPAN::Types qw( Bool Str );
 use Cpanel::JSON::XS qw( decode_json );
 use DateTime;
 
+use MetaCPAN::Script::Mapping::DeployStatement;
+use MetaCPAN::Script::Mapping::Cpan::Author;
+use MetaCPAN::Script::Mapping::Cpan::Distribution;
+use MetaCPAN::Script::Mapping::Cpan::Favorite;
+use MetaCPAN::Script::Mapping::Cpan::File;
+use MetaCPAN::Script::Mapping::Cpan::Mirror;
+use MetaCPAN::Script::Mapping::Cpan::Rating;
+use MetaCPAN::Script::Mapping::Cpan::Release;
+use MetaCPAN::Script::Mapping::User::Account;
+use MetaCPAN::Script::Mapping::User::Identity;
+use MetaCPAN::Script::Mapping::User::Session;
```

Comment:
they are alpha-sorted, what you see here is because of the directory (`Cpan`/`User`).